tests/Shared/WorkloadTesting/BuildEnvironment.cs

```diff
@@ -76,25 +84,16 @@ public BuildEnvironment(bool expectSdkWithWorkload = true, Func<string, string,
                 sdkForWorkloadPath = Path.GetDirectoryName(dotnetPath)!;
             }
 
-            BuiltNuGetsPath = Path.Combine(solutionRoot.FullName, "artifacts", "packages", EnvironmentVariables.BuildConfiguration, "Shipping");
-
-            // this is the only difference for local run but outside-of-repo
-            if (expectSdkWithWorkload)
-            {
-                TestAssetsPath = Path.Combine(AppContext.BaseDirectory, "testassets");
-            }
-            else
-            {
-                TestAssetsPath = Path.Combine(solutionRoot!.FullName, "tests");
-            }
+            BuiltNuGetsPath = Path.Combine(SolutionRoot.FullName, "artifacts", "packages", EnvironmentVariables.BuildConfiguration, "Shipping");
+            TestAssetsPath = Path.Combine(AppContext.BaseDirectory, "testassets");
             if (!Directory.Exists(TestAssetsPath))
             {
                 throw new ArgumentException($"Cannot find TestAssetsPath={TestAssetsPath}");
```

Comment:
When/how does this folder get created? I'm not seeing where it gets created.

Reply:
It contains a copy of the test project:
https://github.com/dotnet/aspire/blob/9990c950c11d0b9b240c94651e48827ad9c26ff1/tests/Aspire.EndToEnd.Tests/Aspire.EndToEnd.Tests.csproj#L27